week1/src/backend/routes/meals.js

```diff
@@ -0,0 +1,12 @@
+const meals = require("../data/meals.json");
+const reviews = require("../data/reviews.json");
+let result =[];
+let mealsWithReviews = meals.map((meal)=>{ 
+    // Find the reviews to the meal
+    let mealReviews = reviews.filter((review) => {
+        return meal.id == review.mealId;
+    })
+    meal.review = mealReviews;
```

Comment:
Since there are multiple:

```suggestion
    meal.reviews = mealReviews;
```